### PR TITLE
remove newMarketsNews variant

### PR DESCRIPTION
**Pull request by leggsimon**
@i-like-robots @aintgoin2goa

### CODE REVIEW

**Comment by i-like-robots** [comment]
+2000

File: views/front-page-n-teaser.html [hunk 1]
```diff
@@ -109,19 +109,8 @@ <h1 class="n-util-visually-hidden">Financial Times Home</h1>
...
-			<div class="" data-o-grid-colspan="12 M6">
-				{{section "technology" @root}}
-			</div>
-			<div class="" data-o-grid-colspan="12 M6">
-				{{section "markets" @root}}
-			</div>
-		</div>
-	{{/if}}
+	{{section "technology" @root}}
+	{{section "markets" @root}}
```
Comment (i-like-robots) [inline views/front-page-n-teaser.html]: OH NO THIS IS WRONG

File: server/sections/technology/presenter.js [hunk 2]
```diff
@@ -12,6 +12,6 @@ const views = {
 module.exports = function technologyPresenter ({flags, edition}){
 	const data = presenter.getData(dataStore, edition);
 	const model = new ViewModel({data, flags});
-	const view = flags.newMarketsNews ? views.fullwidth : views.halfwidth;
+	const view = views.halfwidth;
```
Comment (leggsimon) [inline server/sections/technology/presenter.js]: Yeah, you reckon just completely remove them?

**Comment by i-like-robots** [comment]
+3000

**Pull request merged**